workflows/prognostic_c48_run/runtime/emulator.py

```diff
@@ -45,6 +37,7 @@ class Config:
     # will ignore the emulator for any z larger than this value
     # remember higher z is lower in the atmosphere hence "below"
     ignore_humidity_below: Optional[int] = None
+    mask_kind: str = "default"
```

Comment:
I think it would be more natural to have a `mask_kind=None` be no masking instead of `mask_kind="default", ignore_humidity_below=None`.  See other comment about the state update.

Reply:
The default has the same effect, and always using a mask is nice to avoid adding `if mask_kind is None` anywhere.

Reply:
That's fine.  I was just thinking if using the config, I might have the question, "how do I know masking is off?" or "how do I turn off masking?".  The current default does that, but I don't think it's clear that it's the case.  `mask_kind=None` was an attempt to remedy that, but I think if you just explain in the docstring when masking is "off" that would also suffice.